ydb/docs/en/core/cluster/audit-log-scheme.md

```diff
@@ -0,0 +1,106 @@
+# Schema operations
+
+Schema (or DDL) operation logging makes it possible to monitor changes to schema entities made by users.
+
+Logged operations include:
+- creating, modifying and deleting database objects, tables, directories, connections to external databases and the likes,
+- modifying schema entities access rights (ACL),
+- creating, modifying and deleting users and groups for [login and password authentication](../concepts/auth.md#static-credentials) mode.
+
+## Record attributes
+
+| __Attribute__ | __Description__ |
+|:----|:----|
+| `component` | `schemeshard`.
+| `remote_address` | IP address of the client who sent the request.
+| `subject` | User SID (account name) of the user on whose behalf the operation is performed.
+| `database` | Path of the database in which the operation is performed.
+| `operation` | Operation type.
+| `start_time` | Operation start time.
+| `end_time` | Operation end time.
+| `status` | General operation status: `SUCCESS` or `ERROR`.
+| `detailed_status` | Internal operation status.
+| `tx_id` | Unique transaction identifier.
+
+### Operation specific attributes
+
+Schema operations:
+| __Kind__ | __Attribute__ | __Description__ |
+|:----|:----|:----|
+| _All_ ||
+|| `paths` | List of paths in the database that are changed by the operation (for example, `[/root/db/table-a, /root/db/table-b]`).<br>Required. |
+| _Ownership and permissions_ ||
+|| `new_owner` | User SID of the new owner of the object when ownership is transferred.
+|| `acl_add` | List of added permissions, in [short notation](./short-access-control-notation.md) (for example, `[+R:someuser]`).
+|| `acl_remove` | List of revoked permissions, in [short notation](./short-access-control-notation.md) (for example, `[-R:somegroup]`).
+| _[User (or custom) attributes](../concepts/datamodel/table?#users-attr)_ ||
+|| `user_attrs_add` | List of custom attributes added when creating objects or updating attributes (for example, `[attr1: A, attr2: B]`).
+|| `user_attrs_remove` | List of custom attributes removed when creating objects or updating attributes (for example, `[attr1, attr2]`).
+
+[Credentials](../concepts/auth#static-credentials) management operations:
```

Comment:
Especially  to lessen the confusion a reader might have about operations on schema objects (which are represented as paths) vs user/groups objects (which technically are not a schema objects as they do not have representation as paths).

There are some confusion anyway. May be it would be better (in some future) to move login/group operation into the separate article. Though it could turn out to be content poor and feel somewhat artificial.